packages/dev/core/src/FlowGraph/flowGraph.ts

```diff
@@ -12,37 +16,49 @@ import { FlowGraphEventBlock } from "./flowGraphEventBlock";
  * The graph can then be started, which will init and start all of its event blocks.
  */
 export class FlowGraph {
-    private _blocks: FlowGraphBlock[] = [];
+    /**
+     * The variables defined for this graph
+     */
+    public variableDefinitions: FlowGraphVariableDefinitions = new FlowGraphVariableDefinitions();
+
+    private _eventBlocks: FlowGraphEventBlock[] = [];
     private _sceneDisposeObserver: Nullable<Observer<Scene>>;
+    private _scene: Scene;

```

Comment:
public readonly scene: Scene , and then set it in the constructor. Saves a few lines of a getter :-)